File: app/src/main/java/com/alexanderPodkopaev/dev/behancer/ui/projects/ProjectsActivity.java
package com.alexanderPodkopaev.dev.behancer.ui.projects;

import androidx.fragment.app.Fragment;

import com.alexanderPodkopaev.dev.behancer.common.SingleFragmentActivity;

/**
 * Created by Vladislav Falzan.
 */

public class ProjectsActivity extends SingleFragmentActivity {

    @Override
    protected Fragment getFragment() {
        return ProjectsFragment.newInstance();
    }
}

File: app/src/main/java/com/alexanderPodkopaev/dev/behancer/ui/projects/ProjectsActivity.kt
package com.alexanderPodkopaev.dev.behancer.ui.projects

import androidx.fragment.app.Fragment
import com.alexanderPodkopaev.dev.behancer.AppDelegate
import com.alexanderPodkopaev.dev.behancer.common.SingleFragmentActivity
import com.alexanderPodkopaev.dev.behancer.data.Storage


class ProjectsActivity : SingleFragmentActivity() {
    override val fragment: Fragment
        get() = ProjectsFragment.newInstance()

}